src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests4.cs

```diff
@@ -2960,5 +2960,75 @@ public class MatchFailureException : InvalidOperationException
             CompileAndVerify(compilation, expectedOutput: @"3
 MatchFailureException()");
         }
+
+        [Fact]
```

Comment:
>Fact [](start = 9, length = 4)

nit: note that the current formatter inserts a space between the type and the positional sub-patterns: `A (...`.
You may want to adjust the formatting in these tests.
For reference, `FormatRecursivePattern_SpaceBetweenTypeAndPositionalSubpattern`

Reply:
I will report a bug against the formatter.  There is a design principle of patterns that they have a certain correspondence to construction syntax, and you would not write `new A (1)` to create, so you should not write `A (1)` to match.